testsuite/kuadrant/policy/__init__.py

```diff
@@ -1,9 +1,25 @@
 """Contains Base class for policies"""
 
+from dataclasses import dataclass
+
 from testsuite.kubernetes import KubernetesObject
 from testsuite.utils import check_condition
 
 
+@dataclass
+class CelPredicate:
+    """Dataclass that references CEL predicate e.g. auth.identity.anonymous == 'true'"""
+
+    predicate: str
+
+
+@dataclass
+class CelExpression:
+    """Dataclass that references CEL expression"""
+
+    expression: str
```

Comment:
I'd leave these 2 as a string. Also, they should be under RateLimit module if they are only used there

Reply:
what do you mean by leaving them as string?

Reply:
I don't see the reason to use dataclasses here

Reply:
```
      when:
        - predicate: "request.method == 'POST'" 
```

This is predicate in action. For this you would need to specify "predicate" every time